### DISCUSSION TITLE
about diff-ymd-package

### DISCUSSION THREAD

**Discussion by spirito7phil** [spirito7phil/diff-ymd-package#4] [Q&A] [answered]
what diff-ymd-package provides

**Comment by farhan7reza7** [comment, marked as answer]
diff-ymd-package provides APIs for calculating the difference between two dates in formatted ways like (aY bM cD)(aYears bMonths cDays) or customized desired formats like aY-bM-cD or aYears-bMonths-cDays or kDays or mWeeks or nMonths etc.
